### DISCUSSION TITLE
Hurl parameters

### DISCUSSION THREAD

**Discussion by realtica** [jellydn/hurl.nvim#210] [Q&A] [answered]
### Details

Hello,
How to add parameters to request like: "--insecure" ??

**Comment by sweep-ai-deprecated** [comment]
❌ The ticket to PR workflow is no longer supported. We have moved to a new chat interface in private beta. Please contact us at team@sweep.dev to try it out.

:book: For more information on how to use Sweep, please read our [documentation](https://docs.sweep.dev).



<div align="right"><sup><sup><em>Tracking ID: 7c363dcae4</em></sup></sup></div>

**Comment by jellydn** [comment]
Hi @realtica Could you tell me a bit about your usage? Are you using v1 or v2? Thanks.

**Comment by realtica** [comment]
> Hi @realtica Could you tell me a bit about your usage? Are you using v1 or v2? Thanks.

I am using version 1, and all working well, but I need to include "insecure param" like
`hurl --insecure file.hurl`
is it possible?

**Reply by jellydn** [reply, marked as answer]
If so, you should parse as the option after the command, e.g: `HurlRunAt --insecure`

**Comment by gusilva** [comment]
I recommend using the "hurl" option. For example:

`
GET https://localhost/api/user

[Options]
insecure: true
`

Here the hurl documentation that has this configuration:
https://hurl.dev/docs/request.html#options

**Comment by realtica** [comment]
Thanks @gusilva !
